internal/db/migrations/002_utility_functions.sql

```diff
@@ -62,6 +62,8 @@ BEGIN
               AND table_type = 'BASE TABLE'
               AND table_name !~* '^(schema_version|jobs|job_dependencies|heartbeats)$'
               AND table_name !~* '_reservation_details$'
+              AND table_name !~* 'reservation_instances$'
```

Comment:
This could have been a single line:

```
AND table_name !~* '_reservation_(details|instances)$'
```

Reply:
so it didn't work because we don't have a prefix to the reservation_instances table (like aws_). 
Do you know how can we include that as well?

Reply:
Tho I do not feel like being less verbose is better here, so I'd just keep it as is.
```
AND table_name !~* '_?reservation_(details|instances)$'
```